### PR TITLE
feat(app): add TxRecipientCard component

### LINKED ISSUE #31: Create TxRecipientCard component
## Figma Reference
https://www.figma.com/file/uEckQY4EaQ5rXwyISg6d1g/Fuel-Wallet?node-id=79%3A7832

## Props
- `address?: String`

## Subcomponents
- `TxRecipientCard.Loading`
  - This component is just a loading state using our Skeleton

## Notes
- This component need to be smart sufficient to detect if an address is contract or user

### PR DESCRIPTION
**Pull request by pedronauck**
✨ Fix #31 

## Tasks

- [x] Create component for `account` type
- [x] Create component for `contract` type
- [x] Create component skeleton
- [x]  Add unit tests

### CODE REVIEW

File: packages/app/src/systems/Transaction/components/TxRecipientCard/TxRecipientCard.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+      )}
+      {contract && (
+        <Box css={styles.iconWrapper}>
+          <Icon icon={Icon.is("Code")} size={22} />
+        </Box>
+      )}
+      <Flex css={styles.info}>
+        {contract && <Text css={styles.from}>Contract</Text>}
+        {address && (
+          <Copyable value={address}>{shortAddress(address)}</Copyable>
```
Comment (LuizAsFight) [inline packages/app/src/systems/Transaction/components/TxRecipientCard/TxRecipientCard.tsx:44]: can we make address font size a little bit smaller? figma mockup is little bit smaller and feels better to me

File: packages/app/src/systems/Transaction/components/TxRecipientCard/TxRecipientCard.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+};
+
+export const TxRecipientCard: TxRecipientCardComponent = ({
+  account,
+  contract,
+}) => {
+  const address = account ? account.address : contract?.address;
+  return (
+    <Card css={styles.root}>
+      <Text css={styles.from}>From</Text>
```
Comment (LuizAsFight) [inline packages/app/src/systems/Transaction/components/TxRecipientCard/TxRecipientCard.tsx:26]: can we make From font size a little bit smaller? figma mockup is little bit smaller and feels better to me

File: packages/app/src/systems/Transaction/components/TxRecipientCard/TxRecipientCard.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+          background="fuel"
+        />
+      )}
+      {contract && (
+        <Box css={styles.iconWrapper}>
+          <Icon icon={Icon.is("Code")} size={22} />
+        </Box>
+      )}
+      <Flex css={styles.info}>
+        {contract && <Text css={styles.from}>Contract</Text>}
```
Comment (LuizAsFight) [inline packages/app/src/systems/Transaction/components/TxRecipientCard/TxRecipientCard.tsx:42]: if we remove this extra line, and add in "from" row as parenthesis, maybe would be better?
like this \\/

image

File: packages/app/src/systems/Transaction/components/TxRecipientCard/TxRecipientCardLoader.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+      height={105}
+      viewBox="0 0 92 104"
+      backgroundColor={darkColors.gray2}
+      foregroundColor={darkColors.gray3}
+      {...props}
+    >
+      <rect x="15" width="62" height="14" rx="4" />
+      <rect x="20" y="26" width="52" height="52" rx="26" />
+      <rect x="20" y="26" width="52" height="52" rx="26" />
+      <rect y="90" width="92" height="14" rx="4" />
```
Comment (LuizAsFight) [inline packages/app/src/systems/Transaction/components/TxRecipientCard/TxRecipientCardLoader.tsx:20]: can we change to `y="92"` just to fit a bit better with the loaded component?

**Review by LuizAsFight — CHANGES_REQUESTED** [review]

File: packages/app/src/systems/Transaction/components/TxRecipientCard/TxRecipientCard.tsx (hunk 1 above)
Comment (pedronauck) [reply packages/app/src/systems/Transaction/components/TxRecipientCard/TxRecipientCard.tsx:44]: ✅

File: packages/app/src/systems/Transaction/components/TxRecipientCard/TxRecipientCard.tsx (hunk 2 above)
Comment (pedronauck) [reply packages/app/src/systems/Transaction/components/TxRecipientCard/TxRecipientCard.tsx:26]: ✅

File: packages/app/src/systems/Transaction/components/TxRecipientCard/TxRecipientCard.tsx (hunk 3 above)
Comment (pedronauck) [reply packages/app/src/systems/Transaction/components/TxRecipientCard/TxRecipientCard.tsx:42]: ✅

File: packages/app/src/systems/Transaction/components/TxRecipientCard/TxRecipientCardLoader.tsx (hunk 4 above)
Comment (pedronauck) [reply packages/app/src/systems/Transaction/components/TxRecipientCard/TxRecipientCardLoader.tsx:20]: ✅

**Review by LuizAsFight — APPROVED** [review]
lgtm

**Pull request merged**